### PR TITLE
DEVDOCS-3720 Missing Product Video

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-3720](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3720)

Fixed example not showing up in docs (youtube links don't show up there because of stoplight)

### CODE REVIEW

File: reference/catalog.v3.yml [hunk 1]
```diff
@@ -2673,17 +2673,17 @@ paths:
...
                     video_id:
+                      type: string
                       maxLength: 25
                       minLength: 0
-                      type: string
                       description: |
                         The ID of the video on a host site.
                       x-required:
                         - post
+                      example: 'https://www.youtube.com/watch?v=`R12345677`'
```
Comment (traci707) [inline reference/catalog.v3.yml:2685]: The video is no longer available.

**Review by traci707 — COMMENTED** [review]
@bc-andreadao Can you find a different video to use in the example?

**Comment by bc-andreadao** [comment]
Note: `video_id` needed to be under 25, so I changed up the example

File: reference/catalog.v3.yml (hunk 1 above)
Comment (bc-andreadao) [reply reference/catalog.v3.yml:2685]: I'm having trouble finding a video that's under 25 characters

Comment (bc-andreadao) [reply reference/catalog.v3.yml:2685]: Is it supposed to be a string of numbers?

Comment (traci707) [reply reference/catalog.v3.yml:2685]: @bc-andreadao oh okay, I think I see.

File: reference/catalog.v3.yml [hunk 2]
```diff
@@ -2649,7 +2649,7 @@ paths:
                 The model for a POST to create a video on a product.
               allOf:
                 - title: Product Video Base
-                  type: object
+                  description: Common ProductVideo properties.
```
Comment (markcmurphy) [inline reference/catalog.v3.yml:2652]:
```suggestion
                  description: Common Product Video properties.
```

File: reference/catalog.v3.yml (hunk 1 above)
Comment (markcmurphy) [reply reference/catalog.v3.yml:2685]: @bc-andreadao The `video_id` is the string that comes after the `v` query param in the YouTube link. So if the video URL is `https://www.youtube.com/watch?v=z3fRu9pkuXE`, one would just need to enter `z3fRu9pkuXE`.

Here is a full request body:

```json
{
  "title": "Writing Great Documentation",
  "description": "A video about documenation",
  "sort_order": 1,
  "type": "youtube",
  "video_id": "z3fRu9pkuXE"
}
```

**Review by markcmurphy — CHANGES_REQUESTED** [review]

File: reference/catalog.v3.yml (hunk 2 above)
Comment (bc-andreadao) [reply reference/catalog.v3.yml:2652]: made the suggested change

File: reference/catalog.v3.yml (hunk 1 above)
Comment (bc-andreadao) [reply reference/catalog.v3.yml:2685]: thanks, added in the example

**Review by markcmurphy — APPROVED** [review]
✅✅✅

**Pull request merged**